### DISCUSSION TITLE
Using Mongoose

### DISCUSSION THREAD

**Discussion by sadrajafari** [scenerystack/community#33] [Q&A]
Hi, 
I have installed mongoose library in my project. when I want to use it I have error which is "Uncaught TypeError: The specifier “mongoose” was a bare specifier, but was not remapped to anything. Relative module specifiers must start with “./”, “../” or “/”"

Could you help me with that?

Thanks.

**Comment by brettfiedler** [comment]
Hi @sadrajafari! We're not too sure how to help here off the bat. Could you provide the specific context in which you're importing mongoose and trying to use it alongside SceneryStack code?
